bin/new_component.sh

```diff
@@ -5,4 +5,9 @@ cd "${BASH_SOURCE%/*}"
 
 NAME=$1
 ../node_modules/.bin/ts-node --project ../tsconfig.bin.json ./create_new_component.ts $NAME
+npx jscodeshift --dry-run --componentName="$NAME" -t ./addToSidebar.ts ../docs/components/SideBar/SideBar.jsx
```

Comment:
Not even remotely blocking, but it could be cool to add some simple test cases for these codemods!

Reply:
Yeah! Would love to dive into how to do this properly. I notice you set those up in your repo and it's amazing